spec/index.bs

```diff
@@ -8210,20 +8212,30 @@ dictionary GPUFragmentState
             - [$validating GPUProgrammableStage$]({{GPUShaderStage/FRAGMENT}}, |descriptor|, |layout|) succeeds.
             - |descriptor|.{{GPUFragmentState/targets}}.length must be &le;
                 |device|.{{device/[[limits]]}}.{{supported limits/maxColorAttachments}}.
+            - Let |entryPoint| be [$get the entry point$]({{GPUShaderStage/FRAGMENT}}, |descriptor|).
+            - Let |usesDualSourceBlending| be `false`.
```

Comment:
The current validation code will check that if the pipeline is expecting dual-source blending, the shader should as well but we should also check that if the shader expects dual-source blending, the pipeline does as well.

Reply:
> But we should also check that if the shader expects dual-source blending, the pipeline does as well.

Actually according to [our tests](https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/src/dawn/tests/end2end/DualSourceBlendTests.cpp;l=312) we can write outputs to `@blend_src(1)` while on the API side the blend factor doesn't use `Src1`.

Reply:
The reason I was concerned about it is that HLSL just reuses `SV_Target1` for the 2nd blend src of target 0 (there is no distinction between the blend src and render target index).

But it seems it doesn't matter, it will get ignored either way.

Reply:
> I was concerned about it because HLSL just reuses SV_Target1 for the 2nd blend src of target 0 (there is no distinction between the blend src and render target index).

In current SPEC we only allow using one color attachment when dual source blending is enabled, so we don't need to consider the second color attachment when we use dual source blending.